.gitignore

```diff
@@ -3,5 +3,5 @@ target
 *.gz
 *.out
 com.github.swhkd.pkexec.policy
-*rc
+# *rc
```

Comment:
unnecessary change, can be undone

Reply:
I've fixed this by adding an ignore pattern in #242

Reply:
Thanks @aadilshabier.

Reply:
Um 😅  Should I just remove this file or since it is anyways same as your change, it can just stay?